nlp-core-open/src/main/java/edu/isi/nlp/corpora/gigaword/IndexFlatGigaword.java

```diff
@@ -37,6 +37,7 @@ private static void trueMain(String[] argv) throws IOException {
     final File outputDirectory = params.getCreatableDirectory("rawGigawordOffsetIndexDir");
 
     for (final File sourceDir : gigawordRawRoot.listFiles()) {
+      if (!sourceDir.isDirectory()) {continue;}
```

Comment:
Can you format this as a multi-line `if`? We try to follow https://google.github.io/styleguide/javaguide.html